.github/workflows/testsuite.yaml

```diff
@@ -63,54 +63,51 @@ jobs:
           env_vars: OS,PYTHON
           fail_ci_if_error: true
 
+  # Use pip for diversity
+  # do this on min_deps because it's hard to get mpi4py to install from pip
   min_deps:
     env:
-      ENV_NAME: min_deps
-      PYTHON: 3.9
+      PYTHON: "3.10"
     name: Min Deps Testing
-    defaults:
-      run:
-        # Adding -l {0} helps ensure conda can be found properly.
-        shell: bash -l {0}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@main
         with:
-          fetch-depth: 1
-      - name: Setup Minimamba
-        uses: conda-incubator/setup-miniconda@v2
+          fetch-depth: 0
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
         with:
-          miniforge-variant: Mambaforge
-          miniforge-version: latest
-          use-mamba: true
           python-version: ${{ env.PYTHON }}
-          environment-file: ci/${{ env.ENV_NAME }}.yaml
-          activate-environment: ${{ env.ENV_NAME }}
 
-      - name: Conda Info
+      # need to install the testing packages but not the other optional ones
+      - name: Install
         run: |
-          conda info -a
-          conda list
+          pip install coverage pytest pytest-cov pytest-xdist
+          pip install -e .
+
+      - name: Environment Info
+        run: |
+          pip list
           PYVER=`python -c "import sys; print('{:d}.{:d}'.format(sys.version_info.major, sys.version_info.minor))"`
           if [[ $PYVER != ${{ env.PYTHON }} ]]; then
             exit 1;
           fi
 
-      - name: Install
+      - name: Run Tests
         run: |
-          pip install --no-deps .
+          python -m pytest -n auto --cov=pyuvsim --cov-config=.coveragerc --cov-report xml:./coverage.xml
 
-      - name: Run Tests
+      - name: check coverage report
         run: |
-          python -m pytest -n auto --cov=pyuvsim --cov-config=.coveragerc --cov-report xml:./coverage.xml --junitxml=test-reports/xunit.xml
+          ls
+          cat coverage.xml
```

Comment:
this feels like a bit of a debug item in the action list. Did we want to keep this?

Reply:
no, I think you're right it's a debugging thing